src/system/WatchableEventManagerLibevent.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Derived from `WatchableSocketLibevent.h`; see https://gist.github.com/kpschoedel/451cf083e707c2a5405267dca5384f2a for diff